packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -1,46 +1,534 @@
-/*!
- * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
- */
+/* globals window */
+import util from 'util';
+
+import {includes} from 'lodash';
+import bowser from 'bowser';
+import uuid from 'uuid';
+
+import LoggerProxy from '../common/logs/logger-proxy';
+import {MEETING_ERRORS} from '../constants';
 
-/* eslint-disable */
+import {error, eventType, errorCodes as ERROR_CODE} from './config';
 /**
- *
- *
+ * @description Metrics handles all the call metrics events
  * @export
  * @class Metrics
  */
-export default class Metrics {
+class Metrics {
   /**
-   * Create an instance of Metrics.
-   * @memberof Metrics
-   */
-  constructor() {} // eslint-disable-line
+     * crating Metrics
+     * @constructor
+     * @public
+     * @memberof Meetings
+     */
+  constructor() {
+    if (!Metrics.instance) {
+      this._events = [];
+      this.meetingCollection = null;
+      Metrics.instance = this;
+    }
 
-  /**
-   *
-   * @returns {null}
-   * @memberof Metrics
-   */
-  media() {}
+    return Metrics.instance;
+  }
 
   /**
-   *
-   * @returns {null}
-   * @memberof Metrics
+   * poste call event metrics
```

Comment:
Looks like a copy/paste error. `Set collection of meetings`?

Reply:
_done_